src/EscapeStack.ts

```diff
@@ -0,0 +1,47 @@
+export class EscapeStack {
```

Comment:
Nah this is a separate repo.  All riq rules are fair-game for re-evaluation here.  I use 2 spaces in all my external repos too, so yeah.  May as well do 2.